PSO-Application.md

```diff
@@ -0,0 +1,114 @@
+model taken from the Planetary Software Organization, and eventually
+work to build something that is as robust as the Astropy Project.
+We recognize that there is a large gulf between where we are now,
+and something as comprehensive as Astropy, and the intent is to try
+and slowly and methodically build.
+
+Participation in PlanetaryPy activities is completely voluntary.
+None of the PlanetaryPy TC members are currently funded to work on
+PlanetaryPy.
+
+
+## Metrics
+
+Since the repos currently under the GitHub planetarypy organization
+have always been a loose federation, no consistent or widespread
+user metrics are available.  However, we will use the planetarypy/pvl
+library as an example case.
+
+The `pvl` project, although it has had no commits in more than two
+years, is widely used by planetary science Python developers to
+deal with the pervasive parameter value language (PVL) standard in
+Planetary Data System (PDS) archives, and used in image labels
+written by the Integrated Software for Imagers and Spectrometers
+(ISIS) software.  It is officially listed as 'being used' by more
+than 20 other GitHub projects, and is likely used by many more.
+Its hosting on the Python Package Index (PyPI) indicates that it
+is downloaded 1.2 thousand times per month.
+
+Simple download counts are one metric, but the fact is that planetary 
+science software and analysis needs a `pvl` library as much as the
+Python standard library needs a `csv` and a `json` library.  And it is 
+the goal of the PlanetaryPy Project to provide libraries like `pvl`
+and other data processing libraries (like those provided by `astropy`)
+to the planetary sciences community.
+
+
+## Scope
+
+The PlanetaryPy TC will have sole responsibility and
+discretion over the Planetary Project in the following areas:
+
+* Setting release dates.
+* Release quality standards.
+* Technical direction.
+* Governance process and practices.
+* Contribution process and practices.
+* Maintaining the list of additional Collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between Collaborators and Working Groups.
+
+
+## Governance, Contributions, Code of Conduct
+
+We plan to adopt the Governance model from the Planetary Software Organization.
+We plan to use the same Code of Conduct (based on the Contributor Covenant, version 1.4)
+that the PSO uses, and a Contribution model that is also similar.
+
+
+## Tools
+
+The PlanetaryPy TC currently has ownership (along with the previous owners) of the
+planetarypy GitHub organization.
+
+
+## IP
+
+There is no existing IP policy nor intellectual property concerns.
+
+### Licenses
+
+The existing projects within the planetarypy GitHub organization
+are under a BSD-3-Clause license, and we plan to use the BSD-3-Clause
+license for the PlanetaryPy project.
+
+
+## Members
+
+* Michael Aye (@michaelaye)
+* Ross Beyer (@rbeyer)
+* Andrew Annex (@AndrewAnnex)
```

Comment:
are u sure you want to store members in this document? or better link to TC members?

Reply:
This PSO-application in our repo is currently a draft.  When we're ready to submit it, we need to make a PR on the PSO TSC repo.  I think the idea is that in this document (unlike our Charter where we do link), the TC member list is explicit so that application review isn't subject to external links.